Source/WebCore/Modules/mediacapabilities/MediaCapabilities.cpp

```diff
@@ -183,10 +183,14 @@ static void gatherDecodingInfo(Document& document, MediaDecodingConfiguration&&
     configuration.canExposeVP9 = document.settings().vp9DecoderEnabled();
 #endif
 
+    RefPtr protectedPage = document.protectedPage();
```

Comment:
Nit: per guidelines, this should be:
```suggestion
    RefPtr protectedPage = document.page();
```